src/styles/_catalog.scss

```
@@ -1,9 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  height: 100vh;
+  max-width: 1024px;
   margin: 0 auto;
+  gap: 48px;
+  justify-content: center;
+
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, 200px);
+  }
```

Use `min-width` instead, it looks more obvious, agree?
```suggestion
   @media (min-width: 488px) {
    grid-template-columns: repeat(2, $card-width);
  }

  @media (min-width: 768px) {
    grid-template-columns: repeat(3, $card-width);
  }

  @media (min-width: 1024px) {
    grid-template-columns: repeat(4, $card-width);
  }
```